src/Session.php

```diff
@@ -181,7 +181,7 @@ public function clear_session( $order = null ) {
 	 *
 	 * @return bool Whether there was a session to resume.
 	 */
-	private function resume_session() {
+	public function resume() {
```

Comment:
Since you call this method each time you access the instance of the session class from the main plugin class, and also within the `get_session` method call now, would it make sense to check if the session is already loaded or not? Since if it is, we should not need to set the session data again right?